### PR TITLE
OCLOMRS-354:  Replaced table with textarea on bulk add concepts page

### PR DESCRIPTION
**Pull request by dessHub**
# JIRA TICKET NAME:
[OCLOMRS-354: Implement adding bulk concepts using IDs populated in the text area on bulk add concepts page](https://issues.openmrs.org/browse/OCLOMRS-354)

# Summary:
- Removed react table displaying concepts with a text are which will populate concepts IDs to be added to dictionary concepts.

### CODE REVIEW

**Comment by AGMETEOR** [comment]
@dessHub fix failing coverage threshold and merge conflicts.

File: src/tests/bulkConcepts/containers/BulkConceptsPage.test.js [hunk 1]
```diff
@@ -143,6 +143,35 @@ describe('Test suite for BulkConceptsPage component', () => {
...
+      addToFilterList: jest.fn(),
+      addConcept: jest.fn(),
+      previewConcept: jest.fn(),
+      fetchFilteredConcepts: jest.fn(),
+    };
+    const wrapper = mount(<Router>
+      <BulkConceptsPage {...props} />
+    </Router>);
+    const event = { target: { name: 'Diagnosis, datatype', checked: true } };
+    wrapper.find('#Diagnosis').simulate('change', event);
```
Comment (Duncanian) [inline src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: You should at least have a comparison for the test. i.e `expect`

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: Fixed it

File: src/components/bulkConcepts/bulkConceptList.jsx [hunk 2]
```diff
@@ -81,8 +38,5 @@ BulkConceptList.propTypes = {
   cielConcepts: PropTypes.arrayOf(PropTypes.shape({
     name: PropTypes.string,
   })).isRequired,
-  handleSelect: PropTypes.func.isRequired,
-  conceptLimit: PropTypes.number.isRequired,
```
Comment (dkayiwa) [inline src/components/bulkConcepts/bulkConceptList.jsx]: Is this related to the ticket?

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 3]
```diff
@@ -126,41 +96,41 @@ export class AddBulkConcepts extends Component {
...
-          ) : (
+            ) : (
             <button
               type="button"
               className="btn btn-primary btn-add-all"
               id="btn-add-all"
               onClick={this.handleAddAll}
               disabled={disableButton}
             >
-              <i className="fa fa-plus" /> Add All Selected
```
Comment (dkayiwa) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: Is this related to the ticket.

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 4]
```diff
@@ -126,41 +96,41 @@ export class AddBulkConcepts extends Component {
...
+            <Link
+              to={`/concepts/${type}/${typeName}/${collectionName}/${dictionaryName}/${language}/`}
+              className="btn btn-secondary"
+            >
+              Cancel
+            </Link>
+            {' '}
+            {this.state.cielConcepts.length === 0 ? (
             <button type="button" className="btn btn-primary" id="btn-add-all" disabled={disableButton}>
-              <i className="fa fa-plus" /> Add All Selected
```
Comment (dkayiwa) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: Is this related to the ticket?

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 5]
```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import matchSorter from 'match-sorter';
```
Comment (dkayiwa) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: Is this related to the ticket?

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 4 above)
Comment (dessHub) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: Yes. Part of Darius' comments.

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 3 above)
Comment (dessHub) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: Yes. Part of Darius' comments.

File: src/components/bulkConcepts/bulkConceptList.jsx (hunk 2 above)
Comment (dessHub) [reply src/components/bulkConcepts/bulkConceptList.jsx]: Yes, It was part of the table I removed. So it was no longer necessary.

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 5 above)
Comment (dessHub) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: This was part of the react table filters, so when I removed the table it was no longer required as it raises `unused import` warnings.

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 3 above)
Comment (dkayiwa) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: On the ticket?

Comment (dessHub) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: Yes
